From 5afa001a8dfb97464a1c0e4395966feb1a4153a5 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <hramrach@gmail.com>
Date: Wed, 08 Jun 2011 21:24:48 +0000
Subject: bgo#652045 - Initialize local_only in GtkFileChooserEntry

This has to be done also when we switch to the entry, otherwise completion for
non-local URIs won't work.
---
diff --git a/gtk/gtkfilechooserdefault.c b/gtk/gtkfilechooserdefault.c
index 3e62dc4..2ee7903 100644
--- a/gtk/gtkfilechooserdefault.c
+++ b/gtk/gtkfilechooserdefault.c
@@ -4539,6 +4539,7 @@ location_switch_to_filename_entry (GtkFileChooserDefault *impl)
   /* Configure the entry */
 
   _gtk_file_chooser_entry_set_base_folder (GTK_FILE_CHOOSER_ENTRY (impl->location_entry), impl->current_folder);
+  _gtk_file_chooser_entry_set_local_only (GTK_FILE_CHOOSER_ENTRY (impl->location_entry), impl->local_only);
 
   /* Done */
 
--
cgit v0.9.0.2
